syntax_and_semantics/next.md

```diff
@@ -11,5 +11,36 @@ while a < 5
   end
   puts a
 end
+
 # The above prints the numbers 2, 4 and 5
 ```
+
+`next` can also be used to exit from a block, for example:
+
+```crystal
+def block
+  yield
+end
+
+block do
+  puts 1, 2
```

Comment:
Please make this `puts` print only one value, it feels weird to complicate the example by making readers recall that puts can take multiple arguments and prints them on separate lines.